server/api.go

```diff
@@ -39,10 +39,10 @@ func (p *Plugin) ServeHTTP(_ *plugin.Context, w http.ResponseWriter, r *http.Req
 	router := mux.NewRouter()
 
 	// Routes called by the plugin's webapp
-	router.HandleFunc("/api/v1/legalhold/list", p.listLegalHolds).Methods(http.MethodGet)
-	router.HandleFunc("/api/v1/legalhold/create", p.createLegalHold).Methods(http.MethodPost)
+	router.HandleFunc("/api/v1/legalhold", p.listLegalHolds).Methods(http.MethodGet)
```

Comment:
nit but I think plural makes sense here since we're retrieving all legal holds

```suggestion
	router.HandleFunc("/api/v1/legalholds", p.listLegalHolds).Methods(http.MethodGet)
```